### DISCUSSION TITLE
感觉可以优化掉异常信息的代码框，代码框上面显示的500/404错误信息就足够了

### DISCUSSION THREAD

**Discussion by IshuaWang** [hcjike/halo-theme-dream2.0-plus#40] [Q&A]
对于非程序员来说，那个代码块和里面的信息没什么用，用代码框展示出来反而影响美观

**Comment by mjsoftking** [comment]
等后续有时间，看看抄一抄别人的错误页，基本弄个404和500也够用

**Comment by mjsoftking** [comment]
如果你有好看的页面，也可以提供一下

**Reply by IshuaWang** [reply]
我的意思是就是把异常信息的代码框去掉就可以，其他保持不变就好。。。就这样应该就够了，404，500页面不需要很复杂

**Reply by mjsoftking** [reply]
https://github.com/zsjy/halo-theme-dream2.0-plus/releases/tag/1.3.6.beta1
这个就删了代码部分，也支持设置4xx错误风格，5xx错误没啥资源，只有默认
